src/app/options/qgsuserprofileselectiondialog.cpp

```diff
@@ -104,4 +116,36 @@ void QgsUserProfileSelectionDialog::onAddProfile()
   }
 }
 
+void QgsUserProfileSelectionDialog::onRemoveProfile()
+{
+  QString selectedProfile = selectedProfileName();
+
+  if ( selectedProfile == QStringLiteral( "default" ) )
+  {
+    QMessageBox::warning( this, tr( "Remove profile" ), QString( "Cannot delete default profile!" ), QMessageBox::Ok );
```

Comment:
```suggestion
    QMessageBox::warning( this, tr( "Remove profile" ), tr( "Cannot delete default profile!" ), QMessageBox::Ok );
```

Reply:
My bad, all fixed in cb80c740e4c631c22ff7b2c84f1c3fbbbf09204d